modules/carto/package.json

```diff
@@ -70,7 +70,8 @@
     "@deck.gl/geo-layers": "^9.0.0-beta",
     "@deck.gl/layers": "^9.0.0-beta",
     "@loaders.gl/core": "^4.2.0",
-    "@luma.gl/core": "^9.0.0"
+    "@luma.gl/core": "^9.0.0",
+    "@luma.gl/shadertools": "^9.0.0"
```

Comment:
Not sure if there's a preferred best practice about this for the visgl ecosystem, since there are more peer dependencies involved — but I'll just flag that adding a new peer dependency is technically a breaking change.

The `@deck.gl/core` and `@deck.gl/aggregation-layers` modules already have a production dependency on `@luma.gl/shadertools` though, so this is unlikely to affect anyone in practice.

My vote would be to put `@luma.gl/shadertools` in `dependencies` instead, for now. I've proposed an idea for simplifying this in the future, at:

- https://github.com/visgl/deck.gl/issues/269#issuecomment-2061554022

Reply:
OK, moving to `dependencies`. I've also moved `@luma.gl/core` there too, shout if that's wrong @donmccurdy